lib/ansible/modules/cloud/azure/azure_rm_virtualmachine.py

```diff
@@ -272,19 +272,11 @@
             - If the subnet is in another resource group, specific resource group by C(virtual_network_resource_group).
         aliases:
             - subnet
-    remove_autocreated:
-        description:
-            - "Set this option when removing a VM using state 'absent' to trigger automatic deletion of all autocreated resources."
-            - "Use this option during creation of a VM to make sure all the resources are removed in case of VM creation failure."
-            - "Following resources will be removed: network interface, network security group, public IP address, storage account."
-        type: bool
-        version_added: "2.8"
     remove_on_absent:
         description:
             - "When removing a VM using state 'absent', also remove associated resources"
-            - "It can be 'all' or a list with any of the following: ['network_interfaces', 'virtual_storage', 'public_ips']"
+            - "It can be 'all' or 'all_autocreated' or  a list with any of the following: ['network_interfaces', 'virtual_storage', 'public_ips']"
```

Comment:
can you pls add description on what's `all` mean, what's `all_autocreated` mean? I spent several minutes to think over the difference, and check code to confirm whether my assumption is correct. :)

Reply:
added better description now